### ISSUE TITLE
no-switch-case-fall-through does not handle braces correctly

### ISSUE THREAD

**Issue by sschoeb** [palantir/tslint#2504]
### Bug Report

- __TSLint version__: 4.3.1
- __TypeScript version__: 2.2.1
- __Running TSLint via__:  cli

#### TypeScript code being linted

```ts
  public testfunc(): string {
    let test = 'bla';
    switch (test) {
      case 'bla': {
        return 'Hallo in case';
      }
      case 'blub':
        return 'Hello blub';
    }

    return 'Hallo outside';
  }

```

with `tslint.json` configuration:

```json
    "no-switch-case-fall-through": true,
```

#### Actual behavior
I got an error that there is a break statement expected before the case
src/app/shared/demo.ts[25, 7]: expected a 'break' before 'case'

It works without the braces around case 'bla'

#### Expected behavior
Working with braces.

**Comment by ajafff** [comment]
This is fixed by #2218 and released with tslint v4.5.0
